### PR TITLE
Create the layout for authenticated pages

### PR DESCRIPTION
**Pull request by Ninjaneer87**

### CODE REVIEW

File: styles/theme/lightThemeOptions.ts [hunk 1]
```diff
@@ -5,7 +5,17 @@ const lightThemeOptions: ThemeOptions  = {
   palette: {
     mode: 'light',
     primary: amber,
+    background: {
+      default: "#FFFFFF",
+      paper: "#f5f5f5",
+    },
+    // text: {
```
Comment (Jezda1337) [inline styles/theme/lightThemeOptions.ts]: If we don't need this comments we can remove them

File: components/shared/layout/appbar/menus/SearchBarMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    touched: false,
+  },
+};
+
+const SearchBarMenu = () => {
+  const { inputs, setInputs, formIsValid } = useForm(initialInputs);
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    console.log("Search submitted");
```
Comment (Jezda1337) [inline components/shared/layout/appbar/menus/SearchBarMenu.tsx]: I guess we don't need this one

File: components/shared/layout/drawer/DrawerMobile.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  overflow: "hidden",
+}));
+
+const DrawerMobile = () => {
+  const [hide, setHide] = useState(false);
+  const [y, setY] = useState(0);
+
+  const handleScroll = useCallback(
+    (e: Event) => {
+      console.log("scrolling");
```
Comment (Jezda1337) [inline components/shared/layout/drawer/DrawerMobile.tsx]: this one as well

**Review by Jezda1337 — CHANGES_REQUESTED** [review]
There is couple of comments and consoles logs that we don't need, pls remove them and we can merge 😀

**Comment by Ninjaneer87** [comment]
Nice catch! All done ! B)

**Pull request merged**